src/main/java/carpet/script/api/Auxiliary.java

```diff
@@ -276,6 +276,14 @@ public static void apply(Expression expression)
             ));
         });
 
+        expression.addContextFunction("item_display_name", -1, (c, t, lv)->{
```

Comment:
Why not jut merge this into display_name? Like is there a specific reason?

Reply:
e has to be an entity value there.